news/_posts/2021-04-08-cuberite-spring-2021-newsletter.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: 'Cuberite Newsletter Spring 2021'
+discussion-id: 3306
```

Comment:
You should now be able to create a draft in https://forum.cuberite.org/forum-7.html. The thread ID can then be included here.

Reply:
Ok - and when this PR is merged I release the forum post 💡